storybooknative/src/Button/index.js.flow

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import { Text } from "react-native";
+import * as React from "react";
+
+import { TOKENS } from "./consts";
+
+export type Type =
+  | "primary"
+  | "secondary"
+  | "info"
+  | "success"
+  | "warning"
+  | "critical"
+  | "facebook"
+  | "google";
+
+export type Size = "small" | "normal" | "large";
+
+export type Tokens = $Keys<typeof TOKENS>;
+
+export type Props = {|
+  +children?: React.Element<Text> | string,
+  +onPress?: () => void,
+  +circled?: boolean,
+  +bordered?: boolean,
+  +disabled?: boolean,
+  +block?: boolean,
+  +type?: Type,
+  +size?: Size,
+  +width?: number,
+|};
+
+declare export var StyledButton: React$ComponentType<Props>;
```

Comment:
Is it necessary to `declare export var`? Maybe just a 
```suggestion
export type StyledButton = React$ComponentType<Props>;
```
would be enough.

Reply:
Okay, maybe ignore this comment since they do
```
declare export var StyledButton: React$ComponentType<Props>;

declare export default React$ComponentType<Props>;
```
in the web Orbit Button component.